locale/update_po_files.sh

```diff
@@ -33,6 +33,24 @@ xargs xgettext \
 --package-version='3.0.3' \
 --msgid-bugs-address="audacity-translation@lists.sourceforge.net" \
 --add-location=file -L Lisp -j -o audacity.pot 
+echo ";; Adding resource files to audacity.pot"
+for path in ../res ; do 
+   find $path -name \*.xml 
+done | \
+sed -E 's/\.\.\///g' |\
+xargs xgettext \
+--its=res.its \
+--no-wrap \
+--default-domain=audacity \
+--directory=.. \
+--add-comments=" i18n" \
+--add-location=file  \
+--copyright-holder='Audacity Team' \
+--package-name="audacity" \
+--package-version='3.0.3' \
```

Comment:
Someone sould make the repeated '3.0.3' in this script an argument or take it from an environment variable and make it more current.  Not for this PR.

All it does it make a comment in the generated .pot file out of date.